### PR TITLE
Add an example of ESIP Science-On-Schema markup to the data directory.

### PR DESCRIPTION
**Pull request by nein09**
These are examples for scientific research data sets and data repositories taken from the [guide](https://github.com/ESIPFed/science-on-schema.org/tree/master/guides) developed by the Science-On-Schema ESIP cluster.

I'm happy to try and make up HTML / RDFA / other markup for these, but this working group has been focused on JSON-LD.

### CODE REVIEW

**Comment by nein09** [comment]
@chrisspradling1980 are you talking about this change specifically? I removed the prov elements from the json-ld markup that I am proposing with this pull request because I thought they may be outside the scope of what the schema.org maintainers would like to see in these examples.

**Comment by chrisspradling1980** [comment]
(https://github.com/schemaorg/schemaorg/issues/3086#issue-1198846881)
I was just do the familiarities of NLP or owl programming language that is persistent inside of GitHub. I know that there are repository containers that store chronological to numeric word cabinets. if we could just eliminate terminology that may pertain to programming language as opposed to everyday usage words that may come conflict with some of the NLP understanding I think this would help scientists and the open source community.
you have to forgive me I'm not very familiar with the terms inside GitHub. I do the best that I can but I never went to school for any of this and half the time I don't know what the heck I'm doing or what I'm reading. I'm just trying to make things easier for symbolic information extractor and biomedical technology

**Review by chrisspradling1980 — APPROVED** [review]
this looks fine however, if there's any way that we could get data sets to challenge the discrepancy or to find discrepancies to include challenges which would create markup to markdown work overs. then multiple data sets can be challenged for multi language discrepancies in scientific fields and also in computer science for data sets. I hope this wouldn't be too hard or too much to ask but this is important.

File: data/sdo-soso-dataset-examples.txt [hunk 1]
```diff
@@ -154,19 +153,19 @@ JSON:
 
 PRE-MARKUP:
 
-<!-- An online repository that hosts scientific research data. -->
+An example of an online repository that hosts scientific research data (JSON-LD only).
 
 MICRODATA:
 
-<!-- An online repository that hosts scientific research data. -->
+TODO
```
[the remark this answers is not in the document]
Comment (smrgeoinfo) [reply data/sdo-soso-dataset-examples.txt:146]: What is the point of this comment?

**Review by RichardWallis — CHANGES_REQUESTED** [review]
Some fixes required after [validating](https://validator.schema.org/) :

**First example**

1.  Misspelling of 'application' in script tag
2. Extra comma at end of description

**Second example**

1. No TYPES: definition
2. availableChannel not a valid property for Organization type

**Comment by nein09** [comment]
@RichardWallis thank you! I thought I had run it through a validator but I guess I hadn't, or at least not the right one.

**Review by RichardWallis — APPROVED** [review]

**Pull request merged**